plugins/kubernetes/app/models/kubernetes/role_validator.rb

```diff
@@ -6,7 +6,8 @@ class RoleValidator
     VALID_LABEL_VALUE = /\A[a-zA-Z0-9]([-a-zA-Z0-9.]*[a-zA-Z0-9])?\z/.freeze # also used in js ... cannot use /i
 
     NAMESPACELESS_KINDS = [
-      'APIService', 'ClusterRoleBinding', 'ClusterRole', 'CustomResourceDefinition', 'Namespace', 'PodSecurityPolicy'
+      'APIService', 'ClusterRoleBinding', 'ClusterRole', 'CustomResourceDefinition', 'Namespace', 'PodSecurityPolicy',
```

Comment:
```suggestion
     'ComponentStatus',
'Namespace',
'Node',
'PersistentVolume',
'MutatingWebhookConfiguration',
'ValidatingWebhookConfiguration',
'CustomResourceDefinition',
'APIService',
'TokenReview',
'SelfSubjectAccessReview',
'SelfSubjectRulesReview',
'SubjectAccessReview',
'CertificateSigningRequest',
'ENIConfig',
'PodSecurityPolicy',
'NodeMetrics',
'PodSecurityPolicy',
'ClusterRoleBinding',
'ClusterRole',
'PriorityClass',
'StorageClass',
'VolumeAttachment',
```

Reply:
kubectl api-resources -o wide

Reply:
I would leave out ENIConfig since that's a custom CRD, but the rest :+1:

Reply:
We already work with nonstandard APIGroups (i.e. ValidatingWebhookConfiguration comes from admissionregistration.k8s.io), by the way.  ENIConfig comes from crd.k8s.amazonaws.com API Group.

Reply:
Removed ENIConfig from suggestion.